test/facade_converter_test.ts

```diff
@@ -36,8 +36,23 @@ function getSources(str: string): {[k: string]: string} {
     'angular2/src/core/di/forward_ref.d.ts': `
         export declare function forwardRef<T>(x: T): T;`,
     'angular2/typings/es6-promise/es6-promise.d.ts': `
-        declare class Promise<R> {}
-        declare module Promise {}`,
+      interface Thenable<R> {
+        then<U>(onFulfilled?: (value: R) => U | Thenable<U>,  onRejected?: (error: any) => U | Thenable<U>): Thenable<U>;
+      }
+      declare class Promise<R> implements Thenable<R> {
```

Comment:
It's just the official interface of Promises. I still need to add `then` support. I removed the Thenable interface though.